src/components/utils/jwt.js

```diff
@@ -1,25 +1,23 @@
 import { currentSeconds } from './index'
 
-const parseJwt = (token) => {
-  const base64Url = token.split('.')[1]
-  const base64 = base64Url.replace('-', '+').replace('_', '/')
-  return JSON.parse(atob(base64))
+export const parseJwt = (token) => {
+  let parsedJwt = null
+  try {
+    const base64Url = token.split('.')[1]
+    const base64 = base64Url.replace('-', '+').replace('_', '/')
+    parsedJwt = JSON.parse(atob(base64))
+  } catch {}
```

Comment:
I think there should be a console error in the `catch` statement so that it's clear the JWT wasn't parsed.

Reply:
The reason why I've removed the console.error here is because this function is called by 2 other function so if there is an invalid jwt the integrator sees the console error 3 times and it becomes noisy, especially when you consider that there is also the onError callback being fired.

Reply:
The reason why I didn't put the console.error here is because this function is called by other 2 functions. That means that if an invalid JWT is passed, there will be three errors logged in the console, plus the onError callback. I'm happy to put a log there but think it can become quite noisy for the integrators

Reply:
@kopijunkie @josokinas Thoughts on this point?

Reply:
It won't be - because here you did silence the error. You can either:
a) log error here
b) `catch (e) { throw e }` to allow error to reach other place it can

Reply:
In the end, I delegated error handling to the function that calls `parseJwt`